lib/cashubrew/NUTs/NUT-02/routes.ex

```diff
@@ -9,7 +7,7 @@ defmodule Cashubrew.Nuts.Nut02.Routes do
   The route to query keys part of a specific keyset from the Mint
   """
   def v1_keys_for_keyset_id do
-    Nut01.Routes.v1_keys() <> "/:keyset_id"
+    "/v1/keys/:keyset_id"
```

Comment:
I thought it would be best not to have to depend on nut01 for this route.

Reply:
Why?

The initial route is defined in the Nut01, it's an extension of it